aws/spec/stack.go

```diff
@@ -27,36 +27,40 @@ import (
-	ResourceTypes   []*string `awsName:"ResourceTypes" awsType:"awsstringslice" templateName:"resource-types"`
-	Role            *string   `awsName:"RoleARN" awsType:"awsstr" templateName:"role"`
-	PolicyFile      *string   `awsName:"StackPolicyBody" awsType:"awsfiletostring" templateName:"policy-file"`
-	Timeout         *int64    `awsName:"TimeoutInMinutes" awsType:"awsint64" templateName:"timeout"`
-	Tags            []*string `awsName:"Tags" awsType:"awstagslice" templateName:"tags"`
-	PolicyBody      *string   `awsName:"StackPolicyBody" awsType:"awsstr"`
-	StackFile       *string   `templateName:"stack-file"`
+	_                      string `action:"create" entity:"stack" awsAPI:"cloudformation" awsCall:"CreateStack" awsInput:"cloudformation.CreateStackInput" awsOutput:"cloudformation.CreateStackOutput"`
+	logger                 *logger.Logger
+	graph                  cloud.GraphAPI
+	api                    cloudformationiface.CloudFormationAPI
+	Name                   *string                               `awsName:"StackName" awsType:"awsstr" templateName:"name"`
+	TemplateFile           *string                               `awsName:"TemplateBody" awsType:"awsfiletostring" templateName:"template-file"`
+	Capabilities           []*string                             `awsName:"Capabilities" awsType:"awsstringslice" templateName:"capabilities"`
+	DisableRollback        *bool                                 `awsName:"DisableRollback" awsType:"awsbool" templateName:"disable-rollback"`
+	Notifications          []*string                             `awsName:"NotificationARNs" awsType:"awsstringslice" templateName:"notifications"`
+	OnFailure              *string                               `awsName:"OnFailure" awsType:"awsstr" templateName:"on-failure"`
+	Parameters             []*string                             `awsName:"Parameters" awsType:"awsparameterslice" templateName:"parameters"`
+	ResourceTypes          []*string                             `awsName:"ResourceTypes" awsType:"awsstringslice" templateName:"resource-types"`
+	Role                   *string                               `awsName:"RoleARN" awsType:"awsstr" templateName:"role"`
+	PolicyFile             *string                               `awsName:"StackPolicyBody" awsType:"awsfiletostring" templateName:"policy-file"`
+	Timeout                *int64                                `awsName:"TimeoutInMinutes" awsType:"awsint64" templateName:"timeout"`
+	Tags                   []*string                             `awsName:"Tags" awsType:"awstagslice" templateName:"tags"`
+	PolicyBody             *string                               `awsName:"StackPolicyBody" awsType:"awsstr"`
+	StackFile              *string                               `templateName:"stack-file"`
+	RollbackConfigurations *cloudformation.RollbackConfiguration `awsName:"RollbackConfiguration" awsType:"awsrollbackconfig"`
```

Comment:
Could you try to remove the `awsrollbackconfig` definition entirely, meaning here in the metadata (awsType:"awsrollbackconfig") and in the `setters.go` file above (more playing the role of a stub I guess)?

I am not sure it is needed during the runtime but it should not be according to what you do with the `RollbackConfigurations` field.

Reply:
I tried, unfortunately then `RollbackConfigurations` not being passed to the Update request payload.